src/SFA.DAS.AANHub.Api/Controllers/ProfilesController.cs

```diff
@@ -12,7 +12,7 @@ public class ProfilesController : ActionResponseControllerBase
     private readonly ILogger<ProfilesController> _logger;
     private readonly IMediator _mediator;
 
-    public override string ControllerName => "Profiles";
+    public override string ControllerName => "MemberProfiles";
```

Comment:
Why was this changed?

Reply:
I think this change you asked me to do when we paired up and left it halfway

Reply:
If you see the usage of this variable in the base class, you will know that this cannot be different than the controller name, so this change needs reverting

Reply:
Done